testsuite/features/support/cobbler_test.rb

```diff
@@ -4,12 +4,6 @@
 require 'xmlrpc/client'
 require 'pp'
 
-# ct = CobblerTest.new( "taylor.suse.de" )
-# bool = ct.running?()
-# bool = ct.system_exists( "bla" )
-# any  = ct.system_get_key( "vbox-ug", "uid" )
-# list = ct.get_list( "systems" )
-
 # Class for clobber test
 class CobblerTest
```

Comment:
This methos here, even if not used, are the ones available in the interface for Cobbler, as a facade component, I think it's fine to keep all of them. Same for other namespaces.